src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,71 @@
+import { Todo } from '../../types/Todo';
+import { Status } from '../../types/Status';
+import React from 'react';
+
+type Props = {
+  sortField: Status;
+  setSortField: (field: Status) => void;
+  todos: Todo[];
+  clearCompleted: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  sortField,
+  setSortField,
+  todos,
+  clearCompleted,
+}) => {
+  const oneTodoCompleted = todos.some(todo => todo.completed);
+
+  const activeTodos = todos.reduce(
+    (acc, todo) => (todo.completed ? acc : acc + 1),
+    0,
+  );
```

Comment:
You can use one iteration for get this two value using reduce method (not critical)